client/main.go

```diff
@@ -29,40 +38,62 @@ func New(facility string) (cacher.CacherClient, error) {
 		return fmt.Sprintf("%s:%d", strings.TrimSuffix(addrs[0].Target, "."), addrs[0].Port), nil
 	}
 
-	certURL := os.Getenv("CACHER_CERT_URL")
-	if certURL == "" {
-		auth, err := lookupAuthority("http", facility)
+	var securityOption grpc.DialOption
+
+	useTLS := env.Bool("CACHER_USE_TLS", true)
+
+	// Must be string to check whether or not it has been defined.
+	verifyTLS := env.Get("CACHER_TLS_VERIFY")
+
+	if !useTLS {
+		securityOption = grpc.WithInsecure()
+
+		// Default to old method for backwards compatibility
```

Comment:
this comment is on the wrong line right?

Reply:
Yeah, this is commenting on the conditional itself, I've moved it to the else block where the backwards compatibility code exists.